modules/exploits/windows/local/wmi_persistence.rb

```diff
@@ -0,0 +1,151 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core/exploit/powershell'
+require 'msf/core/post/windows/powershell'
+require 'msf/core/post/file'
+
+class MetasploitModule < Msf::Exploit::Local
+
+  include Msf::Post::Windows::Powershell
+  include Msf::Exploit::Powershell
+  include Post::Windows::Priv
+  include Msf::Post::File
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'             => 'WMI Event Subscription Persistence',
+      'Description'      => %q{
+          This module will create a permanent WMI event subscription to achieve file-less persistence using one
+          of two methods. The USERNAME method will create an event filter that will query the event log for an
+          EVENT_ID_TRIGGER (default: failed logon request id 4625) that also contains a specified USERNAME_TRIGGER.
+          When these criteria are met a command line event consumer will trigger an encoded powershell payload.
+          The INTERVAL method will create an event filter that triggers the payload after the specified CALLBACK_INTERVAL.
+          This module requires administrator level privileges as well as a high integrity process. It is also
+          recommended not to use stageles payloads due to powershell script length limitations.
+        },
+      'Author'           => ['Nick Tyrer <@NickTyrer>'],
+      'License'          => MSF_LICENSE,
+      'Privileged'       => true,
+      'Platform'         => 'win',
+      'SessionTypes'  => ['meterpreter'],
+      'Targets'       => [['Windows', {}]],
+      'DefaultTarget'    => 0,
```

Comment:
Travis is failing because there's not disclosure date here.  It should look like:
 'DisclosureDate' => 'Month DD YYYY',